config/crd/bases/appstudio.redhat.com_integrationtestscenarios.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.1
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
I think it might be better to update to `v0.16.1` version locally by removing(or moving) the `controller-gen` from the `bin` directory in the integration-service and running `make manifests generate` which should automatically install the newer version of the binary based on the [Makefile configuration](https://github.com/konflux-ci/integration-service/blob/88a813cfef413b4a1304cce54ee753f9935b1c66/Makefile#L177).

Reply:
I didn't mean to update this.  Looks like rerunning `make manifests generate` reverted it back to 0.16.1